Jenkinsfile-deploy

```diff
@@ -1,8 +1,48 @@
 library("tdr-jenkinslib")
```

Comment:
The deploy job is now deploying to two different lambdas which the jenkinslib deploy function doesn't do. If it turns out that other things need it, I'll update the jenkinslib job but for now, this has a custom one.